### ISSUE TITLE
Create temp+hum sensor for xiaomi temperature sensor

### ISSUE THREAD

**Issue by AdamWeglarz** [Smanar/Domoticz-deCONZ#5]
Hi,

This is like more change request,
Could you create also temp+hum sensor for xiaomi temp sensors (like the ones created by domoticz using xiaomi gateway H/W)

Adam

**Comment by Smanar** [comment]
Hello,
In fact, I m making 2 (3 with pression one) sensors in domoticz, because deconz manage them as 2 devices, 2 differents ID, 2 differents websocket message, ect ...
It s same for the cube, 1 device for state information and 1 other for analogic value.

It's a problem for you ?

**Comment by AdamWeglarz** [comment]
It is different than is Xiaomi and when you move from gateway you need to change a lot of things when you use temp+hum sensor. That is why I proposed this change. Xiaomi gateway creates 4 devices:
Temperature
temp+hum
Humidity
General (pressure)

**Comment by Smanar** [comment]
Hu ?
I can understand if you want only 1 sensor (who make temp+hum+pressure) because if you have 5 devices like this one it make 15 sensors and it makes it less clear.
But If you already  have Temperature/Humidity/pressure I don't see an utility to have "temp+hum".

And it' will be difficult to implement in the plugin, to make that, I need to detect this kind of sensor, create a fake sensor, with a fake ID, memorise the reals IDs, look for them, memorise them (because deconz manage them one by one, so need to wait for complete information), compile them to update the fake sensor.
It's realy a big hack just for one sensor.

**Comment by deexiz** [comment]
Same problem : I want to migrate from xiaomi gateway and I can't keep history from my devices temp + hygro.

**Comment by Smanar** [comment]
Yep, I know, there is a dzevent script too, better than the LUA script.
You are not the only one with this request, but there is users who want mix the sensor, users who want to separate then, and deconz by it's working mode, works better with separating them.
I can make a special feature for that, but it will be more complicated than the dzevent script and a lot of "hacky", and it was not in planning plan for the moment.

**Comment by pipiche38** [comment]
I think you can keep the history from domoticz. You need to use the feature 'Replace' in Edit of the existing widget
